### ISSUE TITLE
Functions for prompt/message

### ISSUE THREAD

**Issue by Anachron** [shellfire-dev/shellfire#13]
First off, I like this framework!
Secondly, I am missing functions like this:
https://github.com/Anachron/os-install/blob/master/new/helpers/shell.sh#L51
https://github.com/Anachron/os-install/blob/master/new/helpers/shell.sh#L16

Is it possible to make some functions for this too?

**Comment by raphaelcohn** [comment]
Yes, it is. However, there would be some overlap with current functionality to tease out (eg terminal colouring). And the bashisms (eg [[) would need to be changed. Would it work best as another submodule, rather than being in core, do you think?

**Comment by Anachron** [comment]
@raphaelcohn sure! Make it as sub-module, makes  the most sense.
It's always good to have function for message, prompts and such things.

**Comment by raphaelcohn** [comment]
Fancy having a go?
